client/src/annoMatrix/annoMatrix.ts

```diff
@@ -475,26 +470,23 @@ Return cache keys for columns associated with this query.  May return
   /**
    ** Private interfaces below.
    **/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  _resolveCachedQueries(field, queries) {
-    return (
-      queries
-        // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'query' implicitly has an 'any' type.
-        .map((query) =>
-          _whereCacheGet(this._whereCache, this.schema, field, query).filter(
-            // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'cacheKey' implicitly has an 'any' type.
-            (cacheKey) =>
-              cacheKey !== undefined && this._cache[field].hasCol(cacheKey)
-          )
+  _resolveCachedQueries(
+    field: Field,
+    queries: Query[]
+  ): string[] | number[] | [undefined] {
+    return queries
+      .map((query: Query) =>
+        // @ts-expect-error ts-migrate --- suppressing TS defect (https://github.com/microsoft/TypeScript/issues/44373).
+        // Compiler is complaining that expression is not callable on array union types. Remove suppression once fixed.
+        _whereCacheGet(this._whereCache, this.schema, field, query).filter(
+          (cacheKey?: string | number) =>
+            cacheKey !== undefined && this._cache[field].hasCol(cacheKey)
         )
-        .flat()
-    );
+      )
+      .flat();
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'field' implicitly has an 'any' type.
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  async _fetch(field, q) {
+  async _fetch(field: Field, q: Query): Promise<Dataframe | undefined> {
```

Comment:
On the `q` param, see my comment on fetch() about redefininig as `Query | Query[]`

Reply:
As per related comment, updated:

* [query.ts::Query](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/query.ts#L11)
* [annoMatrix.ts::fetch](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L235)
* [annoMatrix.ts::_fetch](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L489)